sdcm/cluster.py

```diff
@@ -317,7 +317,7 @@ def __init__(self, name, ssh_login_info=None, base_logdir=None, node_prefix=None
         # We should disable bootstrap when we create nodes to establish the cluster,
         # if we want to add more nodes when the cluster already exists, then we should
         # enable bootstrap. So addition means not the first set of node.
-        self.is_addition = False
+        self.is_addition = None
```

Comment:
using "is_" prefix in the name of the variable for three state variable is bad. Use "is_" prefix when variable
is boolean